content/pages/enterprise.html

```diff
@@ -0,0 +1,132 @@
+  
+        {% call homepage.feature_detail_item(header="Additional Audit Tracking", icon="fad fa-signs-post") %}
+            If you need longer term storage for your Audit Logging and Analytics information,
+            we can provide that.
+            This is great for compliance requirements.
+        {% endcall %}
+
+        {% call homepage.feature_detail_item(header="Custom Requirements", icon="fad fa-signs-post") %}
+          Have additional needs that we haven't addressed here?
+          We're happy to work with you to provide additional functionality as required for your documentation solution.
+        {% endcall %}
+  
+      </div>
+    </div>
+  </section>
+
+
+  {# Enterprise customer callout #}
+  <section id="enterprise-inquiry">
+    <div class="ui very padded text container">
+      <div class="ui inverted center aligned padded segment">
+
+        {#  Create semantic UI form for inbound sales prospects #}
+
+        <form method="POST" name="fa-form-1" class="ui inverted form"
+          action="https://webhook.frontapp.com/forms/036c4169294f3b04abaa/Jd5Qy6b7AYZHFZ-iHZ9OtRFlYyZqTGhYZEYBmrsrhHaiRaYLlKHF9nqccJg7pewDwFUbEZ0klqlNdSs7BcUAoJO7aPapcC7JQBlYV5S1GZ04oPvbkzVvriejZRY"
+          enctype="multipart/form-data" accept-charset="utf-8">
+            <h2 class="ui dividing header">Enterprise plan inquiry</h2>
+            <p>
+              Our Enterprise plans have a number of additional features that are not available in our standard plans.
+              Let us know what you're looking for and we'll work with you to find the right solution.
+            </p>
+            <div class="field">
+              <label>Name</label>
+              <input type="text" name="name" placeholder="Name">
+            </div>
+            <div class="field">
+              <label>Company Email Address</label>
+              <input type="email" name="email" placeholder="Company Email Address">
+            </div>
+            <div class="field">
+              <label>What type of documentation are you interested in?</label>
+              <select class="ui dropdown" name="documentation-type">
+                <option value="">Select type of documentation</option>
+                <option value="Developer Docs">Developer documentation</option>
+                <option value="User Guides">User guides</option>
+                <option value="API Documentation">API documentation</option>
+                <option value="Internal Docs">Internal team documentation</option>
+                <option value="Public docs">SaaS product documentation</option>
+                <option value="Other docs">Something else!</option>
+              </select>
+            </div>
+            <div class="field">
+              <label>Do you have any specific requirements or questions?</label>
+              <textarea name="message" rows="2"></textarea>
+            </div>
+            <button class="ui button" type="submit">Submit</button>
```

Comment:
You can use an `inverted button` here as well, which looks a bit nicer on the inverted parent block.

Maybe the copy should be a bit more user-centric as well

```suggestion
            <button class="ui inverted button" type="submit">Contact us</button>
```